adaptive_scheduler/configdb_connections.py

```diff
@@ -22,7 +22,7 @@ class ConfigDBInterface(SendMetricMixin):
     """
 
     def __init__(self, configdb_url, telescope_classes, telescopes_file='data/telescopes.json',
-                 active_instruments_file='data/active_instruments.json'):
+                 active_instruments_file='data/active_instruments.json', overrides=None):
```

Comment:
The changes in this file all pertain to being able to override configdb values with a dict provided (which can be read in from a simple json file in the simulation orchestrator)